src/js/view/controls/controls.js

```diff
@@ -101,10 +101,13 @@ export default class Controls extends Events {
         this.logo = this.playerContainer.querySelector('.jw-logo');
 
         const touchMode = model.get('touchMode');
-        
+
         const focusPlayerElement = () => {
-            const floatingElement = model.get('isFloating') ? this.wrapperElement : this.playerContainer;
-            floatingElement.focus();
+            if (model.get('isFloating')) {
+                this.wrapperElement.querySelector('video').focus();
```

Comment:
something to do with the wrapper element stops the `focus` call from properly fixating on the element (tried every manner of timeouts and call ordering). Focusing the video tag removes this issue.